GetPlayersInSegmentSample/get_players_in_segment_api_sample.js

```diff
@@ -0,0 +1,59 @@
+handlers.GetPlayersInSegmentSample = function (args, context) {
+
+    /*
+        Sample code to use the GetPlayersInSegment API to the process player profiles in a Segment.
+        The GetPlayersInSegment API pages through the all the player profiles
+        in the Segment in batches of size 'MaxBatchSize'.
+        API Doc: https://learn.microsoft.com/en-us/rest/api/playfab/server/play-stream/get-players-in-segment?view=playfab-rest
+    */
+
+    var request =  {
+     MaxBatchSize: 1000,
+     SegmentId: "AAAAAAAAA" // provide your SegmentId here
+    }
+
+    // make the first GetPlayersInSegment API call
+    var playersInSegmentResult =  server.GetPlayersInSegment(request);
+    
+    // process until continuation token is not null to get all the profiles in this Segment
+    while (playersInSegmentResult.ContinuationToken != null) 
+    {
+        // get the current batch of player profiles
+        var playerProfiles = playersInSegmentResult.PlayerProfiles;
+
+        // TODO: 
+        // Add logic here to process the above playerProfiles
+
+        // get the next batch of player profiles in the Segment
+        request =  {
+            ContinuationToken: playersInSegmentResult.ContinuationToken,
+            MaxBatchSize: 1000,
+            SegmentId: "AAAAAAAAA" // provide your SegmentId here
+        }        
+        playersInSegmentResult =  server.GetPlayersInSegment(request);
+    }
+};
+
+
+handlers.GetSegmentPlayerCountSample = function (args, context) {
+
+    /*
+        Sample code to use the GetPlayersInSegment API to get the number of players in a Segment
+        API Doc: https://learn.microsoft.com/en-us/rest/api/playfab/server/play-stream/get-players-in-segment?view=playfab-rest
+    */
+
+    var request =  {
+     MaxBatchSize: 1,
```

Comment:
>MaxBatchSize: 1, [](http://example.com/codeflow?start=5&length=16)

I would remove this parameter and let it use the default value. 

If MaxBatchSize == 1 then it will force us to always export the results to storage in case they use the continuation token. If we use the default value then for some smaller segments we can at least skip saving to storage.

Reply:
Should I make it `0` instead ? 
The deault value is `1000`

Reply:
One of the title was looping through until cont. token is null to get the player count. They were using profiles.length in the response inside the loop to calculate the total count. So, I thought we could stop those repetitive calls by giving them this sample

Reply:
Tried `MaxBatchSize:0` but there is a validation in place in the API, it doesn't allow value < 1.

Reply:
I think the sample is ok, but small batch size values are inefficient for this API. Leaving it at the default is probably best.

We could consider adding support for 0 batch size, which would be a cool optimization where we only return a count and no profiles at all. It would require a documentation change, but it would be backward compatible since we know noone is using 0 right now.

Reply:
Got it, removed the MaxBatchSize param.